nodes.py

```diff
@@ -597,4 +598,18 @@ def upscale(self, image, upscale_method, width, height, crop):
     "CLIPLoader": CLIPLoader,
 }
 
-
+def load_custom_nodes():
+    possible_modules = os.listdir("custom_nodes")
```

Comment:
Use something like this for the path so it works even when the user isn't running it from the comfyui directory.
```os.path.join(os.path.dirname(os.path.realpath(__file__)), "custom_nodes")```

Reply:
Thanks for your advice 👍